src/server/auth/server/roles.go

```diff
@@ -171,6 +171,44 @@ func init() {
 		},
 	})
 
+	// Project related roles
+	projectViewer := registerRole(&auth.Role{
+		Name:          auth.ProjectViewer,
+		ResourceTypes: []auth.ResourceType{auth.ResourceType_PROJECT},
```

Comment:
all of these project roles should also be applicable for the CLUSTER resource type